packages/fether-react/src/Onboarding/Onboarding.js

```diff
@@ -49,11 +49,11 @@ class Onboarding extends Component {
 
   render () {
     return (
-      <div>
+      <React.Fragment>
         <Header title={<h1>Terms of Use</h1>} />
 
         <div className='window_content'>
-          <div className='box -padded-extra'>
```

Comment:
consider removing `-padded-extra` from fether-react/src/assets/sass/shared/_box.scss if we're no longer using it